### PR TITLE
fix: update to aegir 31 and others deps

### PR DESCRIPTION
**Pull request by hugomrdias**

### CODE REVIEW

**Comment by robertkiel** [comment]
Thx for creating this PR. The latest update to multicodec breaks PeerId and a few other dependencies due to https://github.com/multiformats/js-cid/blob/master/package.json#L44

File: src/index.js [hunk 1]
```diff
@@ -3,7 +3,7 @@
 const mh = require('multihashes')
 const multibase = require('multibase')
 const multicodec = require('multicodec')
-const { baseTable: codecs } = require('multicodec/src/base-table.js')
+const { baseTable: codecs } = require('multicodec/src/generated-table')
```
Comment (vmx) [inline src/index.js:6]: We should stop using non-public APIs. We broke the world once, I don't want it to break it again.

If there is no code to name mapping in js-multicodec, we should add it properly to the public API. I can have a look later this week.

Comment (hugomrdias) [reply src/index.js:6]: Cids exports code to name?

Comment (hugomrdias) [reply src/index.js:6]: AFAICT it only changed the file name nothing else

File: src/index.js [hunk 2]
```diff
@@ -3,7 +3,7 @@
 const mh = require('multihashes')
 const multibase = require('multibase')
 const multicodec = require('multicodec')
-const { baseTable: codecs } = require('multicodec/src/base-table.js')
+const { nameToCode: codecs } = require('multicodec')
```
Comment (vmx) [inline src/index.js:6]: Having `multicodec` imported twice is a bit strange, but I guess we want to move fast.

**Review by vmx — APPROVED** [review]

**Review by vmx — APPROVED** [review]
Yay, now the public API of js-multicodec is used!

**Pull request merged**